### ISSUE TITLE
In Batch accessibility maps, Prepare and Calculate button should be 2 different one

### ISSUE THREAD

**Issue by greenscientist** [chairemobilite/transition#525]
Having the Prepare button just change it's name to Calculate is confusing and it's hard to see that something has happen. They should be 2 different button, (That are maybe grayed when not available)

**Comment by tahini** [comment]
This is somewhat related to #477. There should be only one button that does the whole process.

**Comment by greenscientist** [comment]
Yeah, that could work. 
The 2 steps are in fact 
1-upload a file
2-then select fields and settings.

**Comment by greenscientist** [comment]
oK, we really need to revamp this UI. As the blue upload button should be uploading, not just analyse the file locally :/

And it's batch processing, we can wait for the full upload before doing the calculation, we are not in a hurry there.
